stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanFlow.kt

```diff
@@ -81,7 +83,9 @@ internal abstract class CardScanFlow(
             ).apply {
                 subscribeTo(
                     imageStream.map {
-                        SSDOcr.cameraPreviewToInput(it.image, it.viewBounds, viewFinder)
+                        SSDOcr.cameraPreviewToInput(
+                            it.image.scaleAndCrop(it.viewBounds.size()), it.viewBounds, viewFinder
+                        )
```

Comment:
Making this change means that we're scaling down the image from the camera before sending it to OCR. This ties the resolution of the image the ML models operate on to the size of the preview frame, which means that differently sized fragments will affect scan performance.

Instead of modifying this here, we should leave the image as the same size, but adjust the `viewBounds` to include the entire size of the preview image. in this case, it would be the aspect ratio of the image, sized to surround the view bounds. Take a look at the `minAspectRatioSurroundingSize` method.

It should look something like this:
```
SSDOcr.cameraPreviewToInput(it.image, minAspectRatioSurroundingSize(it.viewBounds, it.image.aspectRatio).centeredOn(it.viewBounds), viewFinder)
```